### PR TITLE
Added slack and google calendar to README

### PR DESCRIPTION
**Pull request by aiwalter**

### CODE REVIEW

**Review by mloning — DISMISSED** [review]

**Review by fkiraly — CHANGES_REQUESTED** [review]
Great, thanks!

Minor change requests:
* let's resolve the merge conflicts
* why add the "data" to the sub-title? Just makes it more unwieldy and doesn't add content. Would you mind reverting that?

File: README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ sktime is a library for time series analsyis in Python. It provides a unified in
...
-| **Code** |  [![!pypi](https://img.shields.io/pypi/v/sktime?color=orange)](https://pypi.org/project/sktime/) [![!conda](https://img.shields.io/conda/vn/conda-forge/sktime)](https://anaconda.org/conda-forge/sktime) [![!python-versions](https://img.shields.io/pypi/pyversions/sktime)](https://www.python.org/) [![!black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/alan-turing-institute/sktime/main?filepath=examples) |
```
Comment (mloning) [inline README.md:16]: Can we keep the binder badge?

Comment (fkiraly) [reply README.md:16]: ah yes, why did that get removed?

Comment (aiwalter) [reply README.md:16]: seems I missed that in solving the merge conflict. Will add it again

**Review by mloning — APPROVED** [review]
Thanks @aiwalter - LGTM

**Comment by mloning** [comment]
Merged now! 🎉

**Pull request merged**